var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -342,6 +342,9 @@ class Llvm(CMakePackage, CudaPackage):
     patch('no_cyclades.patch', when='@10:12.0.0')
     patch('no_cyclades9.patch', when='@6:9')
 
+    # patch to enable llvm to compile with gcc-11
```

Comment:
This comment doesn't add more info than is already encoded in the `when=` statement so can you remove it or elaborate (e.g. add a link here, or to some related post) - either is fine.

Otherwise this looks great - thanks for the patch!